### ISSUE TITLE
Thaumcraft requires... Agricraft?

### ISSUE THREAD

**Issue by Bewilderer** [Beyond-Reality/Beyond-Reality-Divergence#301]
Is this an issues or am I supposed to need Agricraft to start on Thaumcraft?
![2016-02-20_11 21 54](https://cloud.githubusercontent.com/assets/17357716/13197776/7216c032-d7c5-11e5-8ba0-3cfd4841ae95.png)

**Comment by vagaprime** [comment]
This is kinda an annoying problem. Considering Agricraft requires the teir 2 magic amp. It makes is so that Thaumcraft is now even more late game.

**Comment by choum33** [comment]
there is already a case open for this, please read to avoid doublepost.
check closed one, there is the updated code by Dominance already (included in next update but you can change it by hand easily)

**Comment by choum33** [comment]
but if your player has already unblock thaumcraft you'll need to reset the player profile and archivements to allow them to use thaumcraft devices

**Comment by sapid** [comment]
Specifically, duplicate of #252. Fix is in https://github.com/Beyond-Reality/Beyond-Reality-Divergence/commit/8489e18cab5f5072cbd29c8a67cd878145c17f57 I believe.

**Comment by Dominance** [comment]
Fixed in 1.2.8
